src/components/standard/article.tsx

```diff
@@ -1,37 +1,31 @@
 // ----- Imports ----- //
 
 import React, { ReactNode } from 'react';
+import { css } from '@emotion/core';
+import { palette } from '@guardian/src-foundations';
+import { from, breakpoints } from '@guardian/src-foundations/mq';
 
 import HeaderImage from 'components/shared/headerImage';
-import ArticleSeries from 'components/shared/articleSeries';
-import ArticleHeadline from 'components/standard/headline';
-import ArticleStandfirst from 'components/standard/standfirst';
-import ArticleByline from 'components/standard/byline';
+import Series from 'components/shared/articleSeries';
+import Headline from 'components/standard/headline';
+import Standfirst from 'components/standard/standfirst';
+import Byline from 'components/standard/byline';
 import { CommentCount } from 'components/shared/commentCount'
-import ArticleBody from 'components/shared/articleBody';
+import Body from 'components/shared/articleBody';
 import Tags from 'components/shared/tags';
 import { darkModeCss, articleWidthStyles } from 'styles';
-import { palette } from '@guardian/src-foundations';
-import { from, breakpoints } from '@guardian/src-foundations/mq';
-import { css } from '@emotion/core';
 import { Keyline } from 'components/shared/keyline';
 import { getPillarStyles } from 'pillar';
 import { Article } from 'article';
```

Comment:
Another possible future cleanup - consistently use `export default`?

Reply:
For types?